### PR TITLE
refactor: remove .net fw samples and unify Producer and Consumer samples

### PR DESCRIPTION
**Pull request by filipeesch**

### CODE REVIEW

File: samples/KafkaFlow.Sample/ConsoleLogHandler.cs [hunk 1]
```diff
@@ -8,19 +8,21 @@ public class ConsoleLogHandler : ILogHandler
...
             Console.ForegroundColor = ConsoleColor.Red;
-            Console.WriteLine($"Kafka Error: {message} | Exception: {JsonSerializer.Serialize(ex)}");
+            Console.WriteLine($"\nKafka Error: {message} | Exception: {JsonSerializer.Serialize(ex)}");
             Console.ForegroundColor = ConsoleColor.White;
         }
 
         public void Info(string message, object data)
         {
-            Console.WriteLine($"Kafka Info: {message} | Data: {JsonSerializer.Serialize(data)}");
+            Console.ForegroundColor = ConsoleColor.Yellow;
```
Comment (pmfernandes) [inline samples/KafkaFlow.Sample/ConsoleLogHandler.cs:17]: **Suggestion**: Use another color for info messages

You're already using Yellow for `Warning`.

File: samples/KafkaFlow.Sample/Program.cs [hunk 2]
```diff
@@ -60,17 +51,31 @@ static void Main(string[] args)
...
+            var printConsole = provider.GetService<PrintConsoleProducer>();
             var consumerAcessor = provider.GetRequiredService<IConsumerAccessor>();
 
             while (true)
             {
-                var input = Console.ReadLine();
+                Console.Write("Number of messages to produce, Pause, Resume, or Exit:");
+                var input = Console.ReadLine().ToLower();
+
+                if (int.TryParse(input, out var count))
```
Comment (dougolima) [inline samples/KafkaFlow.Sample/Program.cs:70]: You can put this ' if ' statement inside the switch :
```csharp
case var input when new Regex(@"\d+").IsMatch(input):
```

File: samples/KafkaFlow.Sample/ConsoleLogHandler.cs (hunk 1 above)
Comment (filipeesch) [reply samples/KafkaFlow.Sample/ConsoleLogHandler.cs]: Changed to green

File: samples/KafkaFlow.Sample/Program.cs (hunk 2 above)
Comment (filipeesch) [reply samples/KafkaFlow.Sample/Program.cs]: Changed to something similiar

**Pull request merged**